astronomer/providers/apache/hive/sensors/hive_partition.py

```diff
@@ -10,11 +10,16 @@
 
 class HivePartitionSensorAsync(HivePartitionSensor):
     """
-    Waits for a given partition to show up in Hive.
-    Note: HivePartitionSensorAsync uses implya library instead of pyhive
-    since pyhive is currently unsupported. Refer https://github.com/dropbox/PyHive.
-    Since we use implya library, please set the connection to use the port 10000 instead of 9083
-    The sensor currently supports auth_mechansim='PLAIN' only.
+    .. note:: HivePartitionSensorAsync uses implya library instead of pyhive
+       since pyhive is currently `unsupported <https://github.com/dropbox/PyHive>`_.
+
+       Since we use implya library, please set the connection to use the port ``10000`` instead of ``9083``
+       The sensor currently supports ``auth_mechansim='PLAIN'`` only.
+
+       The library version of hive and hadoop in ``Dockerfile`` should match the remote cluster where the
+       hadoop and hive is running.
+
+    Waits for a given partition to show up in Hive table.
```

Comment:
can you check how the docs are being generated for this block

Reply:
<img width="728" alt="image" src="https://user-images.githubusercontent.com/92459020/165279486-efc29642-7af4-47a5-9ca7-45a19e6abbc6.png">

Reply:
Make the below changes please
- The sync version of this sensor uses `pyhive`, but `pyhive` is currently unsupported
- you can put a hyperlink on the `impyla` and `pyhive`
- a fullstop is missing after 9083
- This sensor currently..

Reply:
Done.
<img width="673" alt="image" src="https://user-images.githubusercontent.com/92459020/165292617-56ce8533-706a-48a8-a131-e168a02e57a7.png">